src/natcap/invest/coastal_blue_carbon/coastal_blue_carbon.py

```diff
@@ -188,941 +194,1521 @@
+        # These are the few sets of tasks that we care about referring to from
+        # the prior year.
+        prior_stock_tasks = current_stock_tasks
+        prior_net_sequestration_tasks = current_net_sequestration_tasks
+
+    # Calculate total net sequestration
+    total_net_sequestration_raster_path = os.path.join(
+        output_dir, TOTAL_NET_SEQ_ALL_YEARS_RASTER_PATTERN.format(
+            suffix=suffix))
+    _ = task_graph.add_task(
+        func=_sum_n_rasters,
+        args=(list(summary_net_sequestration_raster_paths.values()),
+              total_net_sequestration_raster_path),
+        kwargs={
+            'allow_pixel_stacks_with_nodata': True,
+        },
+        dependent_task_list=summary_net_sequestration_tasks,
+        target_path_list=[total_net_sequestration_raster_path],
+        task_name=(
+             'Calculate total net carbon sequestration across all years'))
+
+    # Calculate Net Present Value for each of the transition years, relative to
+    # the baseline.
+    target_npv_paths = {}
+    for transition_year in (
+            sorted(set(transition_years).union(set([final_year])))[1:]):
+        target_npv_paths[transition_year] = os.path.join(
+            output_dir, NET_PRESENT_VALUE_RASTER_PATTERN.format(
+                year=transition_year, suffix=suffix))
+    _ = task_graph.add_task(
+        func=_calculate_npv,
+        args=(summary_net_sequestration_raster_paths,
+              prices,
+              discount_rate,
+              baseline_lulc_year,
+              target_npv_paths),
+        dependent_task_list=summary_net_sequestration_tasks,
+        target_path_list=list(target_npv_paths.values()),
+        task_name=(
+            'Calculate total net carbon sequestration across all years'))
+
+    task_graph.close()
+    task_graph.join()
+
+
+def _calculate_disturbance_volume(disturbance_magnitude_matrix, stock_matrix):
+    """Calculate disturbance volume.
 
+    Args:
+        disturbance_magnitude_matrix (numpy.array): A float32 matrix of the
+            magnitude of a disturbance.  Nodata values must be NODATA_FLOAT32.
+        stock_matrix (numpy.array): A float32 matrix of the stocks present at
+            the time of disturbance.  Nodata values must be NODATA_FLOAT32.
 
-def get_num_blocks(raster_uri):
-    """Get the number of blocks in a raster file.
+    Returns:
+        A numpy float32 matrix of disturbance volume.
+    """
+    disturbed_carbon_volume = numpy.empty(disturbance_magnitude_matrix.shape,
+                                          dtype=numpy.float32)
+    disturbed_carbon_volume[:] = NODATA_FLOAT32
+    valid_pixels = (~numpy.isclose(disturbance_magnitude_matrix,
+                                   NODATA_FLOAT32) &
+                    ~numpy.isclose(stock_matrix, NODATA_FLOAT32))
+    disturbed_carbon_volume[valid_pixels] = (
+        disturbance_magnitude_matrix[valid_pixels] *
+        stock_matrix[valid_pixels])
+    return disturbed_carbon_volume
+
```

Comment:
Since this is a [top-level function in the module, wouldn't there be two lines](https://www.python.org/dev/peps/pep-0008/#blank-lines)?  FWIW, `pycodestyle` isn't complaining, but I did some light linting in 1fc4864f